packages/react-native/ReactCommon/react/renderer/textlayoutmanager/platform/android/react/renderer/textlayoutmanager/TextLayoutManager.cpp

```diff
@@ -201,6 +202,36 @@ LinesMeasurements TextLayoutManager::measureLines(
   return lineMeasurements;
 }
 
+LinesMeasurements TextLayoutManager::measureLines(
```

Comment:
nit: not super happy that now we end up with both `TextLayoutManager::doMeasureLines` and `TextLayoutManager::measureLines`. Maybe we can just inline them?

Reply:
Done in https://github.com/facebook/react-native/pull/45102/commits/08c8d8dfffe2240c8a02f07527f03a21cd9c753b. I was going for consistency between this and `measure`/`doMeasure`.